### PR TITLE
Add generate model yaml macro

### PR DESCRIPTION
**Pull request by jtalmi**
I add a macro which takes in a model name and returns yml. Right now, I added default empty-string descriptions to model and columns, but I can take that out.

### CODE REVIEW

**Comment by jtalmi** [comment]
Just realized I forgot to use toyaml...going to refator :)

**Comment by jtalmi** [comment]
Hm it looks like pyyaml doesn't support indentation for lists, which is how a lot of people tend to structure their yaml in dbt projects. When I run this:

```
{% set my_dict = {"models": [{"name": "model", "description": "",  "columns": [{"name": "col1", "description": ""}]}]} %}
{% set my_yaml_string = toyaml(my_dict) %}
{% do log(my_yaml_string, info=True) %}
```

It returns:

```
models:
- name: model
  description: '""'
  columns:
  - name: col1
    description: '""'
```
Whereas i'd want a 2 space indent at the beginning of every list item (e.g. "  - name: ...")

Known issue: https://github.com/yaml/pyyaml/issues/234
Possible workaround: https://stackoverflow.com/questions/25108581/python-yaml-dump-bad-indentation

**Comment by clrcrl** [comment]
Hey @jtalmi — this looks really cool! I will take a look later on this week.

Agree that my preference would be indentation on lists. Let me chat internally and I'll let you know about the best step forward :)

**Comment by jtalmi** [comment]
Sounds good @clrcrl . I just realized that I should increase the version to 0.0.2 in this PR.

**Comment by clrcrl** [comment]
Naw, the version number doesn't get meaningfully used by dbt, so we don't bother keeping it in sync!

File: integration_tests/tests/test_generate_model_yaml.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+%}
+
+{% set expected_model_yaml %}
+version: 2
+
+models:
+  - name: model_yaml
+    tables:
+      - name: codegen_integration_tests__data_source_table
+        descriptions: ""
```
Comment (clrcrl) [inline integration_tests/tests/test_generate_model_yaml.sql:13]:
```suggestion
        description: ""
```

**Comment by jtalmi** [comment]
hey @clrcrl circling back here after a while, are there any other changes you'd like me to make? i'm not that familiar with github but i can't see any right now

**Comment by clrcrl** [comment]
Haha I did not realize I had write access on your repo, and thought I was pushing this to one of my branches. Anyway I just rebased and pushed in order to kick the tests off :)

**Pull request merged**